modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -4,81 +4,62 @@ import ReactDOM from 'react-dom';
 import {
   createSubcomponent,
   createElemPropsHook,
-  StyledType,
   useWindowSize,
   useForkRef,
-  getTheme,
 } from '@workday/canvas-kit-react/common';
 import {usePopupModel, usePopupStack} from '@workday/canvas-kit-react/popup';
-import {keyframes} from '@emotion/react';
-import styled from '@emotion/styled';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {Box, BoxProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {useModalModel} from './hooks';
+import {createStencil, cssVar, keyframes} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
 
 export interface ModalOverlayProps extends BoxProps {}
 
-const fadeIn = keyframes`
-  from {
-    background: none;
-  }
-  to {
-    background: rgba(0,0,0,0.65);
-  }
-`;
+const fadeIn = keyframes({
+  '0%': {
+    background: 'none',
+  },
+  '100%': {
+    background: cssVar(system.color.bg.overlay),
+  },
+});
 
-const Container = styled(Box)<StyledType>(
-  {
+export const containerStencil = createStencil({
```

Comment:
is there a better name for this? `modalOverlayContainerStencil`?

Reply:
Yeah probably. I'm updating now